### PR TITLE
Fix Fabric 7 package and dependency versions

### PR DESCRIPTION
**Pull request by ecraig12345**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

Bump all package versions to reflect the major release:
* Packages that successfully published 7.0.0 => 7.0.1
* 6.x.x => 7.0.0
* Some 0.x.x packages => 7.0.0 where it makes sense
* Other 0.x.x packages that aren't ready to release => bump minor version by 100 (in case we need to publish fixes out of the fabric-6 branch)
* Use office-ui-fabric-core 10 everywhere

Also re-enable `ensureConsistentVersions` and fix mismatches that were introduced while it was disabled.

And re-delete the dashboard package, which was re-introduced by a bad merge...

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9433)

### CODE REVIEW

File: packages/foundation/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uifabric/foundation",
-  "version": "0.8.0",
+  "version": "0.109.0",
```
Comment (JasonGore) [inline packages/foundation/package.json]: Just curious, is this basically 100 release of "breathing room" for 6.0 branch publishes?

Comment (ecraig12345) [reply packages/foundation/package.json]: Yes. We could choose a smaller number if you prefer.

Comment (ecraig12345) [reply packages/foundation/package.json]: I don't think anyone expects there to be 100 releases out of the 6.0 branch, but it seemed better to give an excessive amount of space than a conservative amount then have to deal with it potentially running out.

Comment (JasonGore) [reply packages/foundation/package.json]: I was just guessing on the rationale here and wanted to confirm 😄

**Pull request merged**